packages/web-components/src/keplr-connection/KeplrConnection.js

```diff
@@ -0,0 +1,26 @@
+// @ts-check
+import { makeLeader } from '@agoric/casting';
+import { makeImportContext } from '@agoric/wallet/api/src/marshal-contexts';
+import { getKeplrAddress } from './getKeplrAddress';
+import { getChainId } from './getChainId';
+import { watchWallet } from './watchWallet';
+
+const DEFAULT_NETWORK_CONFIG = 'https://ollinet.agoric.net/network-config';
```

Comment:
Deserves a TODO:
```suggestion
// TODO: We need a way to detect the appropriate network-config, and default it to mainnet.
const DEFAULT_NETWORK_CONFIG = 'https://ollinet.agoric.net/network-config';
```

Reply:
Done. What were you thinking by "detect" here? I was thinking the dapp could just be responsible for providing it.